pkg/cmd/server/start/node_args.go

```diff
@@ -37,6 +38,9 @@ type NodeArgs struct {
 	ImageFormatArgs    *ImageFormatArgs
 	KubeConnectionArgs *KubeConnectionArgs
 	CertArgs           *CertArgs
+
+	PodManifestPath    string
+	FileCheckFrequency time.Duration
```

Comment:
If we're not binding these and exposing them on the command line, we don't need to add them as args